components-core/src/main/java/com/adyen/checkout/components/ComponentViewType.kt

```diff
@@ -6,6 +6,6 @@
  * Created by josephj on 1/9/2022.
  */
 
-package com.adyen.checkout.components.ui.view
+package com.adyen.checkout.components
```

Comment:
Why can't his stay in `ui-core`?

Reply:
For `ActionHandlingDelegate` to use it and that class has to be in `components-core`, because it has dependencies on other components classes